<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMainWindow</class>
 <widget class="QMainWindow" name="NewMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>700</height>
     </rect>
    </property>
    <widget class="QPushButton" name="albums_page_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Albums</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tracks_page_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Tracks</string>
     </property>
    </widget>
    <widget class="QPushButton" name="artists_page_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Artists</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settings_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>620</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stacked_widget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>1080</width>
      <height>670</height>
     </rect>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
